### ISSUE TITLE
Increase overall mouse speed

### ISSUE THREAD

**Issue by RZME** [powerbot/RSBot-API#2172]
Mouse speed < 40 is ridiculously slow, not human-like.
Apply the same scaling but have 40 as the new minimum speed

**Comment by AshGreh** [comment]
Do you want it to scale from 40-100 or do you want me to give you something like cruise control?

**Comment by RZME** [comment]
40-140 basically, so set minimum higher and have higher speed max

**Comment by AshGreh** [comment]
@RZME i'm not willing to go beyond the current max speed, sorry. 10 is as fast as it gets. Will 40-100 work for you then?

**Comment by RZME** [comment]
@AshGreh sure

**Comment by AshGreh** [comment]
Update:
I have the formula sort of framed for translation to paris units, It's currently (int) (100 - (36 + Math.ceil((s - 1) * 0.54)));
The issue is currently with from paris units. It's a bit in accurate due to scaling round off.

**Comment by Paris** [comment]
Any quick fix to improve the speed resolution @AshGreh or should I have a relook at the whole formula?

**Comment by AshGreh** [comment]
Nah, this formula is suitable. The round off in accuracy cannot be avoided. If the user wants an accurate speed feedback, it makes sense to have another variable, but that's it.

**Comment by RZME** [comment]
The trade-off between a more human speed range and the slight improvement in accuracy leans more towards the human speed range tbh. So I'm fine with the round-off :) 👍

**Comment by AshGreh** [comment]
Cool! I will have it up by Sunday then.
